rmw_fastrtps_shared_cpp/src/rmw_security_logging.cpp

```diff
@@ -111,8 +113,14 @@ bool apply_logging_configuration_from_file(
   const std::string & xml_file_path,
   eprosima::fastrtps::rtps::PropertySeq & properties)
 {
+  const auto stripped_xml_file_path =
+      std::regex_replace(xml_file_path, std::regex("file://"), "");
```

Comment:
None of this should be necessary anymore. The underlying code has been refactored a bit to be more testable and this is no longer an issue.